pkgs/misc/uboot/default.nix

```diff
@@ -1,3 +1,9 @@
+###! # U-Boot
+###! Bootloader for Embedded boards based on PowerPC, ARM, MIPS and several other processors
+###!
+###! # Note on aarch64 sunxi devices
+###! These devices require BL31 (Trusted Arm Firmware) and SCP (Power Manager) declared as environmental variables that point to the required file for the build to succeed and work without issues, refer to https://u-boot.readthedocs.io/en/latest/board/allwinner/sunxi.html for details
+
```

Comment:
Can you explain why  this change was needed?

 - What's `###!` all about?
 - We don't add package descriptions like that, it's lost into nothingness, the comment is not used at any place. We have a place for that, it's `meta.description`.
 - That note about Allwinner, while meaning well, is not at the correct place. And also is kinda wrong. It should be in the Nixpkgs manual, in a (new) section about packaging U-Boot, if needed.